### ISSUE TITLE
Major install issue with 1.4.0-beta.1

### ISSUE THREAD

**Issue by cyrilletuzi** [angular/angular-cli#7421]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
cli 1.4.0-beta.1
node 8.3.0
npm 5.3.0
macos

### Repro steps.
`npm install @angular/cli@1.4.0-beta.1`
`ng new test`

Same with a `npm cache clean`.

### The log given by the failure.
In `devDependencies` of `package.json` : 
`"@angular/cli": ""` is empty

The installed version of the CLI in `node_modules` is 1.3.1, resulting in all generation commands failing as schematics are missing.

It may or may not be related, but when creating the project : 
- before, displaying the files created was immediate, now bash is freezing for a few seconds
- npm installation is running twice : 
```
Installing packages for tooling via npm.
Installed packages for tooling via npm.
Successfully initialized git.
Installing packages for tooling via npm.
Installed packages for tooling via npm.
```

### Desired functionality.
Should install the correct packages.

**Comment by cexbrayat** [comment]
I confirm.

There are also other differences in the generated project.
See https://github.com/cexbrayat/angular-cli-diff/compare/1.4.0-beta.0...1.4.0-beta.1

We lost a few changes, probably not on purpose, like ignoring `yarn-error.log`, excluding `node_modules` for linting, etc...

**Comment by Brocco** [comment]
I'll take a look at this and get a PR up soon.

**Comment by cyrilletuzi** [comment]
Fixed in 1.4.0-beta.2

**Comment by cexbrayat** [comment]
@Brocco version is back 👍 but we still lost a few minor things in the migration to schematics.

See https://github.com/cexbrayat/angular-cli-diff/compare/1.4.0-beta.0...1.4.0-beta.2

Maybe that was on purpose, but I'm not sure.
